### PR TITLE
Adding links and more details to the mode submission flow

### PR DESCRIPTION
**Pull request by daisukeiot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,18 +7,62 @@ Related tools, samples, and specs can be found in the [Azure/iot-plugandplay-mod
 
 ## Submit a model
 
+1. Create a GitHub account if you do not have one yet : [https://github.com/join](https://github.com/join).
+
+    Learn more about GitHub, please visit [Introduction to Git](https://docs.microsoft.com/en-us/learn/modules/intro-to-git/)
```
Comment (rido-min) [inline README.md:12]: remove lang from docs urls

File: README.md [hunk 2]
```diff
@@ -7,18 +7,62 @@ Related tools, samples, and specs can be found in the [Azure/iot-plugandplay-mod
 
 ## Submit a model
 
+1. Create a GitHub account if you do not have one yet : [https://github.com/join](https://github.com/join).
```
Comment (rido-min) [inline README.md:10]: nit, rm whitespace before semicolon

File: README.md [hunk 3]
```diff
@@ -7,18 +7,62 @@ Related tools, samples, and specs can be found in the [Azure/iot-plugandplay-mod
...
+
+    Learn more about GitHub, please visit [Introduction to Git](https://docs.microsoft.com/en-us/learn/modules/intro-to-git/)
+
+1. Please sign [Contributor License Agreement](https://cla.opensource.microsoft.com/Azure/iot-plugandplay-models?pullRequest=93)
+
 1. Fork the public GitHub repo: [https://github.com/Azure/iot-plugandplay-models](https://github.com/Azure/iot-plugandplay-models).
+
+    Learn more about forking a repo : [https://docs.github.com/en/github/getting-started-with-github/fork-a-repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo)
```
Comment (rido-min) [inline README.md:18]: whitespace before semicolon.

Comment (rido-min) [inline README.md:18]: For consistency I will use the title in the link text instead the URL, like in Introduction to Git.

File: README.md [hunk 4]
```diff
@@ -7,18 +7,62 @@ Related tools, samples, and specs can be found in the [Azure/iot-plugandplay-mod
...
+    By forking and cloning the public GitHub repo, a copy of repo will be created in your GitHub account and a local copy is created in your dev machine.  Please use this local copy to make modifications.
+
+1. Author a new device model with an unique ID using [Digital Twin Model Identifier](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier).  
+
+    Review the [PR requirements](pr-reqs.md) for naming conventions.
+
+1. Save the device model JSON file to a local folder.  
+
+    E.g.
+    C:\iot-plugandplay-models\MyThermostat.json
```
Comment (rido-min) [inline README.md:31]: use backticks for the path

File: README.md [hunk 5]
```diff
@@ -7,18 +7,62 @@ Related tools, samples, and specs can be found in the [Azure/iot-plugandplay-mod
...
 1. Clone the forked repo. Optionally create a new branch to keep your changes isolated from the `main` branch.
-1. Add the new interfaces to the `dtmi` folder using the folder/filename convention. See the [add-models](#import-a-model-to-the-dtmi-folder) tool below.
+
+    By forking and cloning the public GitHub repo, a copy of repo will be created in your GitHub account and a local copy is created in your dev machine.  Please use this local copy to make modifications.
+
+1. Author a new device model with an unique ID using [Digital Twin Model Identifier](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier).  
+
+    Review the [PR requirements](pr-reqs.md) for naming conventions.
+
+1. Save the device model JSON file to a local folder.
```
Comment (rido-min) [inline README.md:28]: suggestion to link to the DTDL VSCode extension, to help users creating new interfaces

File: README.md [hunk 6]
```diff
@@ -7,18 +7,62 @@ Related tools, samples, and specs can be found in the [Azure/iot-plugandplay-mod
...
 
 The PR triggers a series of GitHub actions that will validate the new submitted interfaces, and make sure your PR satisfies all the checks.
 
 Microsoft will respond to a PR with all checks in 3 business days.
 
+**GitHub Operation Workflow**
+
+```text
++-------------------------------------------------+
+| iot-plugandplaymodel repo (Microsoft)           |
```
Comment (rido-min) [inline README.md:47]: nit, its iot-plugandplay-models (with dashes and plural)

**Review by rido-min — CHANGES_REQUESTED** [review]
review links/texts.

File: README.md (hunk 2 above)
Comment (daisukeiot) [reply README.md]: Updated to use consistent style

[Commit](https://github.com/daisukeiot/iot-plugandplay-models/commit/46d2c845e6a831cd4a039aa2492f3a1e921000a4)

File: README.md (hunk 1 above)
Comment (daisukeiot) [reply README.md]: Updated to use consistent style

[Commit](https://github.com/daisukeiot/iot-plugandplay-models/commit/46d2c845e6a831cd4a039aa2492f3a1e921000a4)

File: README.md (hunk 3 above)
Comment (daisukeiot) [reply README.md]: Updated to use consistent style

[Commit](https://github.com/daisukeiot/iot-plugandplay-models/commit/46d2c845e6a831cd4a039aa2492f3a1e921000a4)

File: README.md (hunk 5 above)
Comment (daisukeiot) [reply README.md:31]: Updated to use consistent style

[Commit](https://github.com/daisukeiot/iot-plugandplay-models/commit/46d2c845e6a831cd4a039aa2492f3a1e921000a4)

File: README.md (hunk 3 above)
Comment (daisukeiot) [reply README.md]: Updated to use consistent style

[Commit](https://github.com/daisukeiot/iot-plugandplay-models/commit/46d2c845e6a831cd4a039aa2492f3a1e921000a4)

File: README.md (hunk 4 above)
Comment (daisukeiot) [reply README.md]: Updated to use consistent style

[Commit](https://github.com/daisukeiot/iot-plugandplay-models/commit/46d2c845e6a831cd4a039aa2492f3a1e921000a4)

File: README.md (hunk 6 above)
Comment (daisukeiot) [reply README.md]: Updated to use consistent style

[Commit](https://github.com/daisukeiot/iot-plugandplay-models/commit/46d2c845e6a831cd4a039aa2492f3a1e921000a4)

**Review by rido-min — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -7,18 +7,67 @@ Related tools, samples, and specs can be found in the [Azure/iot-plugandplay-mod
...
+    > [!TIP]  
+    > [DTDL Editor for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.vscode-dtdl) could help you with the language syntax (including auto-completion) and also validate the syntax with DTDL v2.
+
+1. Save the device model JSON file to a local folder.  
+
+    E.g.
+    `C:\iot-plugandplay-models\MyThermostat.json`
+
 1. Validate the models locally using the `dmr-client` tool to [validate](#validate-models).
+1. Add the new interfaces to the `dtmi` folder using the folder/filename convention. See the [add-models](#import-a-model-to-the-dtmi-folder) tool below.
```
Comment (digimaun) [inline README.md:37]: I think we should change the language referencing `add-models`, this was carry over from when we were running with js scripts. 

Something like `". See the dmr-client tool import functionality below."`

**Review by digimaun — APPROVED** [review]

**Pull request merged**